prov/gni/src/gnix_fabric.c

```diff
@@ -363,14 +366,19 @@ static int gnix_getinfo(uint32_t version, const char *node, const char *service,
 				goto err;
 			}
 
-			if (hints->ep_attr->tx_ctx_cnt > 1) {
+			if (hints->ep_attr->tx_ctx_cnt > GNIX_SEP_MAX_CNT) {
```

Comment:
no. let's have the user fix their code right away instead of realizing later they didn't get what they asked for.